test/test_related_posts.rb

```diff
@@ -3,11 +3,9 @@
 class TestRelatedPosts < JekyllUnitTest
   context "building related posts without lsi" do
     setup do
-      stub(Jekyll).configuration do
-        Jekyll::Configuration::DEFAULTS.merge({'source' => source_dir,
+      config = Jekyll::Configuration::DEFAULTS.merge({'source' => source_dir,
                                                'destination' => dest_dir})
-      end
-      @site = Site.new(Jekyll.configuration)
+      @site = Site.new(config)
```

Comment:
This can all be simplified down to:

```ruby
@site = fixture_site
```

it uses `source_dir` and `dest_dir` by default.